package.json

```diff
@@ -4,7 +4,8 @@
   "description": "H5P Editor Course Presentation ==========",
   "main": "index.js",
   "scripts": {
-    "start": "webpack --watch",
+    "start": "npm run watch",
+    "watch": "webpack --watch",
```

Comment:
Har ingen problemer med dette, men vil gjerne vite hvorfor det er lurt?

Reply:
For at editor-cp og cp skal ligne mer. I cp har `npm run watch` vært måten å starte dev mode på, så ville ikke skape forvirring.